src/test/testng-travis.xml

```diff
@@ -28,68 +29,68 @@
+            <class name="teammates.test.cases.browsertests.AdminEmailLogPageUiTest" />
+            <class name="teammates.test.cases.browsertests.AutomatedSessionRemindersTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackConstSumOptionQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackConstSumRecipientQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackContributionQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackMcqQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackMsqQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackNumScaleQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackRubricQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.FeedbackRankQuestionUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorEditStudentFeedbackPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorFeedbackEditCopyUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorFeedbackEditPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorFeedbackPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorFeedbackResultsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorFeedbackSubmitPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorStudentListPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorStudentRecordsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorSubmissionAdjustmentUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorHomePageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorSearchPageUiTest" />
+            <class name="teammates.test.cases.browsertests.StudentCourseDetailsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.StudentFeedbackSubmitPageUiTest" />
+            <class name="teammates.test.cases.browsertests.StudentFeedbackResultsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.StudentProfilePageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorCourseDetailsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.InstructorCommentsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.StudentCommentsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.AdminEmailPageUiTest" />
+            <class name="teammates.test.cases.browsertests.AdminSessionsPageUiTest" />
+            <class name="teammates.test.cases.browsertests.AdminSearchPageUiTest" />
```

Comment:
Yes, definitely not in this PR. 
Other things being equal, it's better to follow the convention i.e. of putting tests in the same package. Unfortunately, now we have to consider the cost of changing. Probably now is not a good time to do it.